python/doc/theory/probabilistic_modeling/copulas.rst

```diff
@@ -3,53 +3,58 @@
 Copulas
 -------
 
-| In this part, we will define the concept of copula.
+Let :math:`F` be a multivariate distribution function of dimension
+:math:`d` whose marginal distribution functions are
+:math:`F_1,\dots,F_d`. There exists a copula
+:math:`C: [0,1]^d \mapsto [0,1]` of dimension :math:`d` such that for :math:`\vect{x}\in \Rset^d`, we have:
```

Comment:
```suggestion
:math:`\inputDim` whose marginal distribution functions are
:math:`F_1,\dots,F_\inputDim`. There exists a copula
:math:`C: [0,1]^\inputDim \mapsto [0,1]` of dimension :math:`\inputDim` such that for :math:`\vect{x}\in \Rset^\inputDim`, we have:
```

Reply:
In that PR, I introduced the math_notations in the section "Reliability and sentitivity". It would be too much to do all the other files at once. I propose to accept this PR and then to continue section by section.
Do you agree?

Reply:
I don't see the problem with accepting this suggestion (and the others of this kind). All it takes is to click on the button...

Reply:
And to change the others, you could do it quite quickly by searching in this file for the strings `_d`, `{d}`,  `^d` and 
```
`d`
```

Reply:
No I don't agree.
@joseph: if I accept the suggestion of Michael, some files in tha section will ba cieherent with math_nottaions and the otherones not. It will be confusing for the reader.
@mbaudin47 : math_notations do not concern only the input dimension ....

I really prefer that we do these modifications in a new PR that will follow this one very soon.